sql-plugin/src/main/java/com/nvidia/spark/rapids/GpuColumnVector.java

```diff
@@ -69,6 +69,14 @@ public static synchronized void debug(String name, ai.rapids.cudf.ColumnVector c
     }
   }
 
+  private static String hexString(byte[] bytes) {
+    StringBuilder str = new StringBuilder();
```

Comment:
nit: could create with capacity here to avoid reallocations

Reply:
nit: we can use [Hex.encodeHexString(bytes)|](https://commons.apache.org/proper/commons-codec/archives/1.10/apidocs/index.html)